jdaviz/core/loaders/resolvers/resolver.py

```diff
@@ -294,6 +400,10 @@ def __init__(self, *args, **kwargs):
         # Listen for changes to app.state.settings and update traitlet
         self.app.state.add_callback('settings', self._on_app_settings_changed)
 
+    def vue_link_by_wcs(self, *args):
+        self.app._jdaviz_helper.link_data(align_by='wcs')
```

Comment:
this won't work for deconfigged - can we go through the plugin instead?  And can we add one test that tests in deconfigged (or maybe just transition the tests to go through it since anything in deconfigged should also work in Imviz now)?

The plugin should exist as long as image data is already loaded - if an image is NOT loaded, then we probably should disable everything in this PR (but it seems to currently still show the prompt to link by WCS).

```suggestion
        plg = self.app._jdaviz_helper.plugins.get('Orientation', None)
        if plg is not None:
            plg.align_by = 'WCS'
```

Reply:
https://github.com/user-attachments/assets/ccb6db44-cc25-4cd2-b8a4-d3848e841d1f